tests/geometry_test_cases.py

```diff
@@ -1696,6 +1697,41 @@ def test_matrix_innerproduct_and_embedded_innerproduct_coincide(
         """
         # Not yet implemented due to need for local basis implementation
 
+    def test_dist_vectorization(self, space, n_samples):
```

Comment:
As discussed, these tests are not needed as long as you make PullbackDiffeoMetricTestCase inherit from RiemannianMetricTestCase.

--> Do this and let's see what happens, typically which tests we might need to skip?